.travis.yml

```diff
@@ -1,10 +1,23 @@
 language: node_js
+addons:
```

Comment:
are these used to test webgl in a server env?

Reply:
Yes, this is added because we have`gl` in devDependencies. And we run headless-gl test in travis